backbone.js

```diff
@@ -79,6 +79,9 @@
         return _[method](this[attribute], value);
       };
       case 3: return function(iteratee, context) {
+        if (predicateTransform && !_.isFunction(iteratee) && _.isObject(iteratee)) {
```

Comment:
Indeed it would be helpful to have (like underscore):
```js
collection.filter('user');
collection.map('user');
```
for
```js
collection.filter(function isUserDefined(model) {
  return !_.isUndefined(model.get('user')); // or isFalsy ?
});
collection.map(function getUser(model) {
  return model.get('user');
});
```
I'm working on it.